cabal.project

```diff
@@ -64,7 +64,7 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-shell
-  tag: 33e4d04fafbfe094422a45203dda0900be109439
+  tag: bfb61481e6cdae61a8c203f53fdea06d36e8ab83
```

Comment:
`2044172cd8856e927860e9b5c731a4343462206e` or `bfb61481e6cdae61a8c203f53fdea06d36e8ab83`? This isn't consistent with the nix or stack files.

Reply:
Dunno how that happened...